packages/cli/cli.js

```diff
@@ -144,12 +147,55 @@ yargs.command(
 yargs.command(
     "disable-tracking",
     "Disable tracking of Webiny stats.",
-    () => {},
+    () => {}, // eslint-disable-line
     async () => {
         await require("./sls/disableTracking")();
     }
 );
 
+yargs.command(
+    "scaffold <name>",
+    "Scaffold your application with one of the available templates. (Run it from within your app's folder)",
+    yargs => {
+        yargs.positional("name", {
+            describe: "The name to be assigned to your generated template."
+        });
+    },
+    async argv => {
+        const regexIsCliModule = /(^@webiny\/cli-scaffold-.+$)|(^webiny-cli-scaffold.*$)/;
+        const moduleToChoice = module => `${module.scaffold.name} - ${module.scaffold.description}`;
+
+        const webinyModulesNames = (await fs.readdir("../node_modules/@webiny")).map(
+            moduleName => `@webiny/${moduleName}`
+        );
+        const nodeModulesNames = await fs.readdir("../node_modules");
+        const scaffoldModulesNames = webinyModulesNames
+            .concat(nodeModulesNames)
+            .filter(moduleName => moduleName.match(regexIsCliModule));
+        const scaffoldModules = scaffoldModulesNames.reduce(
+            (modules, crtModuleName) => modules.concat(require(crtModuleName)),
+            []
+        );
+        const choices = scaffoldModules.map(module => moduleToChoice(module));
+        if (choices.length === 0)
+            throw new Error(
+                "Oddly, there are no available scaffold templates. This must be a coding error..."
+            );
+        const { scaffoldTemplate } = await inquirer.prompt({
+            type: "list",
+            name: "scaffoldTemplate",
+            message: "Pick the template you'd like to employ",
+            choices
+        });
+        const = scaffoldModules.find(
+            module => moduleToChoice(module) === scaffoldTemplate
+        );
+        selectedScaffoldModule.scaffold.generate({
+            serviceName: argv.name
+        });
```

Comment:
Once you have the template, you should get options from it and run them through `inquirer`, so each templte can have its own wizard. `argv.name` is not really necessary as we don't know what the user wants to create until he selects a template. Let's do that per-template for now; basically, let every template handle its own wizard.

Reply:
As mentioned, `scaffold` will be flexible and allow users to write their own plugins (via `webiny-cli-scaffold-*` packages), which means they might write a scaffold that's not related to services at all.

I agree with @Pavel910, I believe `webiny scaffold` will suffice, then users select what they want via the inquirer.